pkg/apimachinery/utils/meta.go

```diff
@@ -20,6 +23,8 @@ const AnnoKeyUpdatedTimestamp = "grafana.app/updatedTimestamp"
 const AnnoKeyUpdatedBy = "grafana.app/updatedBy"
 const AnnoKeyFolder = "grafana.app/folder"
 const AnnoKeySlug = "grafana.app/slug"
+const AnnoKeyBlob = "grafana.app/blob"
```

Comment:
blob not required for this PR, but including it does not cause any issues and including it here will simplify merge conflicts for the next PR adding blob store suppport